ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/blocks/blockbody/versions/capella/BlindedBeaconBlockBodySchemaCapellaImpl.java

```diff
@@ -0,0 +1,206 @@
+    implements BlindedBeaconBlockBodySchemaCapella<BlindedBeaconBlockBodyCapellaImpl> {
+
+  private BlindedBeaconBlockBodySchemaCapellaImpl(
+      final String containerName,
+      NamedSchema<SszSignature> randaoReveal,
+      NamedSchema<Eth1Data> eth1Data,
+      NamedSchema<SszBytes32> graffiti,
+      NamedSchema<SszList<ProposerSlashing>> proposerSlashings,
+      NamedSchema<SszList<AttesterSlashing>> attesterSlashings,
+      NamedSchema<SszList<Attestation>> attestations,
+      NamedSchema<SszList<Deposit>> deposits,
+      NamedSchema<SszList<SignedVoluntaryExit>> voluntaryExits,
+      NamedSchema<SyncAggregate> syncAggregate,
+      NamedSchema<ExecutionPayloadHeader> executionPayloadHeader,
+      NamedSchema<SszList<BlsToExecutionChange>> blsToExecutionChanges) {
+    super(
+        containerName,
+        randaoReveal,
+        eth1Data,
+        graffiti,
+        proposerSlashings,
+        attesterSlashings,
+        attestations,
+        deposits,
+        voluntaryExits,
+        syncAggregate,
+        executionPayloadHeader,
+        blsToExecutionChanges);
+  }
+
+  public static BlindedBeaconBlockBodySchemaCapellaImpl create(
+      final SpecConfigCapella specConfig,
+      final AttesterSlashingSchema attesterSlashingSchema,
+      final String containerName) {
+    return new BlindedBeaconBlockBodySchemaCapellaImpl(
+        containerName,
+        namedSchema(BlockBodyFields.RANDAO_REVEAL, SszSignatureSchema.INSTANCE),
+        namedSchema(BlockBodyFields.ETH1_DATA, Eth1Data.SSZ_SCHEMA),
+        namedSchema(BlockBodyFields.GRAFFITI, SszPrimitiveSchemas.BYTES32_SCHEMA),
+        namedSchema(
+            BlockBodyFields.PROPOSER_SLASHINGS,
+            SszListSchema.create(
+                ProposerSlashing.SSZ_SCHEMA, specConfig.getMaxProposerSlashings())),
+        namedSchema(
+            BlockBodyFields.ATTESTER_SLASHINGS,
+            SszListSchema.create(attesterSlashingSchema, specConfig.getMaxAttesterSlashings())),
+        namedSchema(
+            BlockBodyFields.ATTESTATIONS,
+            SszListSchema.create(
+                new AttestationSchema(specConfig), specConfig.getMaxAttestations())),
+        namedSchema(
+            BlockBodyFields.DEPOSITS,
+            SszListSchema.create(Deposit.SSZ_SCHEMA, specConfig.getMaxDeposits())),
+        namedSchema(
+            BlockBodyFields.VOLUNTARY_EXITS,
+            SszListSchema.create(
+                SignedVoluntaryExit.SSZ_SCHEMA, specConfig.getMaxVoluntaryExits())),
+        namedSchema(
+            BlockBodyFields.SYNC_AGGREGATE,
+            SyncAggregateSchema.create(specConfig.getSyncCommitteeSize())),
+        namedSchema(
+            BlockBodyFields.EXECUTION_PAYLOAD_HEADER, new ExecutionPayloadHeaderSchema(specConfig)),
+        namedSchema(
+            BlockBodyFields.BLS_TO_EXECUTION_CHANGES,
+            SszListSchema.create(
+                new BlsToExecutionChangeSchema(), specConfig.getMaxBlsToExecutionChanges())));
```

Comment:
We're already creating an instance of this in `SchemaDefinitionsCapella` (via `SignedBlsToExecutionChange`) and probably should only have one instance of it so probably need to pass that schema in like we do for `AttesterSlashingSchema` given we'll want to be able to get the `SignedBlsToExecutionChange` schema from schema definitions for gossip stuff I think.